faq.md

```diff
@@ -130,7 +130,35 @@ A: This depends on your internet bandwidth, where your data is located, your int
 
 Q: How should I keep track of my tissue samples and associated data? Is there an identifier schema I need to follow?
 
-A: Please follow the sample identifier standard operating procedure: [HTAN Identifiers SOP](https://bit.ly/htan_ids).
+A: Please follow the sample identifier standard operating procedure: [HTAN Identifiers SOP](https://bit.ly/htan_ids) and the [RFC HTAN Identifiers](https://docs.google.com/document/d/1K6jjWQdvtGRKUMmjC7jvt69hd3CbnSyryVWpNJVndDA/edit?pli=1#). 
+
+Each HTAN entity will be uniquely identified by an HTAN identifier.
+
+HTAN will support two formats for biological identifiers: one for research participants, and the other for biospecimens and data-files (both of which are considered derived entities).
```

Comment:
@ychae I see what you mean when saying wordy... but I think for now this makes sense. Perhaps we can think about how we can focus on examples in the FAQ that capture most common cases: e.g. we could include links to spreadsheets that have example data and identifiers